ftp-protocol/get-and-put-files.md

```diff
@@ -0,0 +1,55 @@
+---
+description: Instructions for using FTP commands to access and transfer files using NetExec.
+

```

Comment:
I think here is `---` missing. Atleast the page doesnt get added to the gitbook when i try to preview the change